src/website/shared/management/commands/ingest_delta_cve.py

```diff
@@ -126,4 +128,9 @@ def handle(self, *args: Any, **kwargs: Any) -> None:
             next_ingestion + datetime.timedelta(days=x)
             for x in range((date - next_ingestion).days + 1)
         ):
-            ingest_day(repo, day)
+            try:
+                ingest_day(repo, day)
+            except UnknownObjectException:
```

Comment:
This is a thing that GitHub can throw, and we're catching that exception type in `ingest_day`. Shouldn't we instead catch *any* exception and print that rather than swallowing those details?

Reply:
If there's more fatal errors, there's no need to continue the loop, the script should terminate early and this is abnormal.

If there's a _known_ non-problematic error, we should catch it and decide whether to continue or not. Here, we decide that if GitHub has not a release on a single day, no big deal, we shall continue.